src/types.ts

```diff
@@ -29,6 +29,27 @@ interface UniqueOptions<T extends string> {
   timestamp: "inserted_at" | "scheduled_at";
 }
 
+type ReplacementOption =
+  | "args"
+  | "max_attempts"
+  | "meta"
+  | "priority"
+  | "queue"
+  | "scheduled_at"
+  | "tags";
```

Comment:
这里得用 camel case ，在发送请求之前去转换一下比较好。因为我们的代码里面的 key 用的是 camel case
```suggestion
  | "args"
  | "maxAttempts"
  | "meta"
  | "priority"
  | "queue"
  | "scheduledAt"
  | "tags";
```

Reply:
你把这个抽出来写一下测试呢
https://github.com/Byzanteam/jet-queue-plugin-js/blob/e53210bfc6f2827758611a81e13d743851291045/src/queue.ts#L250

Reply:
还有中做法，是 enqueue 的后端来兼容，你们商量一下呢

cc @c01dcat

Reply:
暂时还是 sdk 这边转换吧，要做兼容的话，其他地方也应该统一兼容，比如 `unique` 里 `timestamp` 的 `inserted_at` 和 `scheduled_at` 等